deploy/eck-stack/examples/elasticsearch/hot-warm-cold.yaml

```diff
@@ -0,0 +1,199 @@
+---
+eck-elasticsearch:
+  nodeSets:
+  - name: masters
+    count: 1
+    config:
+      node.roles: ["master"]
+      # Comment out when setting the vm.max_map_count via initContainer, as these are mutually exclusive.
+      # For production workloads, it is strongly recommended to increase the kernel setting vm.max_map_count to 262144
+      # and leave node.store.allow_mmap unset.
+      # ref: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-virtual-memory.html
```

Comment:
```suggestion
      # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-virtual-memory.html
```

Reply:
I haven't created suggestions for all occurrences but there are others, same for `k8s-volume-claim-templates.html`.